src/main/java/se/kth/meta/db/Dbao.java

```diff
@@ -10,6 +10,7 @@
 import javax.persistence.EntityManager;
```

Comment:
What exactly is the use of this class? If seems to serve as a facade? If so, you should implement it using the same scheme as in the rest of the app. 
Consider e.g. se.kth.bbc.jobs.jobhistory.JobHistoryFacade as an example. (Stateless bean extending AbstractFacade.)